client/src/ui/utils/ShowToast.tsx

```diff
@@ -0,0 +1,31 @@
+import { Bounce, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+// Utility function to show toast
+export const showToast = (message: string, type: "success" | "error") => {
```

Comment:
Probably cleaner to just make two `showSuccessToast` and a `showErrorToast` functions instead? i don't think this would need to be dynamic at least